wofs/wofs_app.py

```diff
@@ -183,7 +187,7 @@ def make_wofs_tasks(index, config, year=None, **kwargs):
         time = Range(datetime(year=year[0], month=1, day=1), datetime(year=year[1] + 1, month=1, day=1))
 
     extent = {}
-    if 'x' in kwargs and kwargs['x'] is not None:
+    if 'x' in kwargs and kwargs['x']:
```

Comment:
`x` being 0 is a valid value.

Reply:
`x` in this case is a tuple. Was trying to avoid the empty tuple sent by click from equating to True.